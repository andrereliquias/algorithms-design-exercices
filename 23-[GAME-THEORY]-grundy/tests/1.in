10
7043
3054
8448
7617
2545
1239
4163
7269
1579
9471
