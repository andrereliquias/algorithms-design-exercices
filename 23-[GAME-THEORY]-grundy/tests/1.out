Teco
Tico
Tico
Teco
Tico
Teco
Tico
Tico
Tico
Teco
